easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-intel-2018b.eb

```diff
@@ -0,0 +1,28 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'hunspell'
+version = '1.7.0'
+
+homepage = 'http://hunspell.github.io/'
+description = """Hunspell is a spell checker and morphological analyzer library and program designed for languages
+ with rich morphology and complex word compounding or character encoding."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/hunspell/hunspell/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20180311')]
```

Comment:
and add binutils here if we move it to GCCcore

Reply:
as above. I tried it with GCCcore-8.2.0, 7.3.0, and 7.2.0 with the same failure.

Reply:
Your error message shows it used `/usr/bin/ld.gold` which leads me to believe that 'binutils' '2.31.1' wasn't added ad a builddep when you tried this.